inheritance/src/main/java/inheritance/Restaurant.java

```diff
@@ -1,17 +1,37 @@
 package inheritance;
 
-public class Restaurant {
+import jdk.internal.org.objectweb.asm.tree.InsnList;

```

Comment:
This breaks your code, I do not think you are using it so you should delete this line